.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "dokuwiki/shared"]
 	path = dokuwiki/shared
-	url = git://git.php.net/web/shared.git
```

Comment:
Why?

Reply:
Because it works without auth, the current one didn't.

Reply:
That URL does not require auth, only the (different) push url does.  Were you having problems using that URL?

Reply:
I remember that I had issues with it during installation on the new wiki server, don't remember the exact error message.